### PR TITLE
[5698] Upgrade react-intl

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5698

### CODE REVIEW

**Comment by rart** [comment]
Either this is the wrong title/description for the PR or branch got mixed up with the react-intl update

**Comment by jvega190** [comment]
There was a mix up, fixed name/description of PR

File: ui/app/src/state/epics/content.ts [hunk 1]
```diff
@@ -382,6 +382,7 @@ const content: CrafterCMSEpic[] = [
                   });
             } else {
               return showConfirmDialog({
+                // @ts-ignore - Type '(siteId: string, path: string) => Observable<any>' is not assignable to type 'FormatXMLElementFn<string, string>' Error.
```
Comment (rart) [inline ui/app/src/state/epics/content.ts]: This looks like a legitimate error

File: ui/app/src/components/PublishOnDemandWidget/PublishOnDemandWidget.tsx [hunk 2]
```diff
@@ -426,13 +421,31 @@ export function PublishOnDemandWidget(props: PublishOnDemandWidgetProps) {
...
             <div className={classes.noteContainer}>
               <Typography variant="caption" className={classes.note}>
-                {formatMessage(mode === 'studio' ? messages.publishStudioNote : messages.publishGitNote, {
-                  a: (msg) => (
-                    <Link key="Link" href="#" onClick={toggleMode} className={classes.noteLink}>
-                      {msg}
-                    </Link>
-                  )
-                })}
+                {mode === 'studio' ? (
```
Comment (rart) [inline ui/app/src/components/PublishOnDemandWidget/PublishOnDemandWidget.tsx:424]: Please test

**Review by rart — CHANGES_REQUESTED** [review]
See notes

**Review by rart — APPROVED** [review]

**Pull request merged**